src/layouts/MainLayout.js

```diff
@@ -90,14 +85,10 @@ const MainLayout = ({ data = {}, children, pageContext }) => {
               padding-bottom: ${contentPadding};
             `;
 
-            return nav?.id === rootNav.id ? (
+            return (
               <animated.div style={style} css={containerStyle}>
                 <RootNavigation nav={nav} />
```

Comment:
Can we remove `Subnavigation` component entirely since it won't be used? Or were you waiting on doing that until we get better sense of rest of work involved?

Reply:
i was going to leave that in case but we probably could!